.github/workflows/scale-test-100-gce.yaml

```diff
@@ -217,6 +220,8 @@ jobs:
           export CL2_PROMETHEUS_SCRAPE_CILIUM_OPERATOR=true
           export CL2_PROMETHEUS_SCRAPE_CILIUM_AGENT=true
           export CL2_PROMETHEUS_MEMORY_SCALE_FACTOR=2.0
+          export CL2_LOAD_TEST_THROUGHPUT=50.0
+          export CL2_DELETE_TEST_THROUGHPUT=50.0
```

Comment:
FYI, depending on how much metrics we fetch from cilium-agent, we might want to tune the scrape interval:
https://github.com/kubernetes/perf-tests/pull/2651/files#diff-6a4d7ca416707ad8bf902645ddddde590e8fde0c2711e67ee7e68ea38e593d61R3

I've set it to 30s by default, but I am pretty sure it won't be enough.